changes/tests.py

```diff
@@ -1942,6 +1952,76 @@ def test_baby_switch_whatsapp(self):
             'type': 11,
         })
 
+    @responses.activate
+    def test_baby_switch_momconnect_whatsapp(self):
+        """
+        If the mother is subscribed to momconnect whatsapp, then when
+        switching to baby messages, they should receive those through
+        whatsapp as well.
+        """
+        # Pretest
+        self.assertEqual(Registration.objects.all().count(), 0)
+        # Setup
+        # make registrations
+        self.make_registration_whatsapp_pmtct_prebirth()
```

Comment:
should this be creating a normal prebirth subscription or a pmtct prebirth subscription?